test_suite/package/test_otel.py

```diff
@@ -0,0 +1,132 @@
+import pytest
+import logging
+import time
+import json
+
+from test_suite.generic.test_generic import TestsSubscriptionManager
+logger = logging.getLogger(__name__)
+
+
+@pytest.fixture()
+def check_instance_status(request, instance_data, host):
```

Comment:
Why do we need this fixture?
The instance will already be operative and ssh-ready at the moment we run this test suite.

Reply:
In order to make changes in the security of the instance it is required that the instance status and system_status will be passed/ok status, otherwise it will not be possible to make this change.

Reply:
Got it! I thought that the status of the instance was already ok after CIV deployed it.
Out of curiosity: Do we know how much time does it take for the instance to be "ready" for security changes, after we deploy it?

Reply:
mmm it takes about 2-3 minutes